docs/de-DE/config/README.md

```diff
@@ -426,9 +426,10 @@ vicmd_symbol = "[V](bold green) "
 
```

Comment:
Sorry I should have mentioned this earlier but can you remove the changes to non English documentation. All our translations are done using Crowdin, https://translate.starship.rs/, and it gets confused if the translations are changed outside of Crowdin. P.S we are always looking for people to help translate if you want to help out.

Reply:
No problem. Updated the PR